Hearthstone Deck Tracker/Controls/Card.xaml.cs

```diff
@@ -19,5 +19,21 @@ public bool HasTooltip
 			get => (bool)GetValue(HasTooltipProperty);
 			set => SetValue(HasTooltipProperty, value);
 		}
+
+		private void Rectangle_ToolTipOpening(object sender, System.Windows.Controls.ToolTipEventArgs e)
+		{
+			TooltipCardImage.SetCardId(CardId());
+		}
+
+		public string CardId()
+		{
+			var card = DataContext as Hearthstone.Card;
+			return card?.Id;
+		}
```

Comment:
Can be simplified as
```suggestion
		public string CardId => (DataContext as Hearthstone.Card)?.CardId;
```